roles/monitoring_sap/tasks/main.yml

```diff
@@ -28,6 +29,23 @@
   when: sap_monitoring_action == "add"
   block:
 
+    - name: Ensure required variables are defined
```

Comment:
Rename task: Assert required variables are defined for adding SAP monitoring configuration